Felienne/hedy
# [FEATURE] Adds admin achievement statistics

**Description**
In this PR we implement a new feature to get more insights into the achievements distribution of our users. We create an overview within the admin dashboard to show the total amount of achievements as well as the percentage of users achieving this. These values are always calculated on runtime, but as it is the admin page the extra server load with not be that much. (As only one user can view it)

**Fixes**
This PR fixes #2719

**How to test**
Make sure you are an admin and navigate to the admin dashboard `/admin`. Verify that there is a new button called "Achievement statistics". Also verify that when visiting this page all achievements are shown, accompanied by a count and percentage of users having this one achieved.

![Schermafbeelding 2022-05-16 om 11 09 44](https://user-images.githubusercontent.com/38155424/168559464-1f774611-ba8e-4685-9b96-ed88f273c72d.png)
